.github/actions/install-go-and-dependencies/action.yml

```diff
@@ -0,0 +1,23 @@
+name: InstallGoAndDependencies
+description: 'Installs Go Downloads and installs Karpenter Dependencies'
+runs:
+  using: "composite"
+  steps:
+    - run: sed -En 's/^go[[:space:]]+([[:digit:].]+)$/GO_VERSION=\1/p' go.mod >> $GITHUB_ENV
+      shell: bash
```

Comment:
What's this bit for? Is there a different default shell?

Reply:
That's just required in composite actions.